redirects.json

```diff
@@ -171,30 +95,10 @@
     "fromPath": "/nft/",
     "toPath": "/en/nft/"
   },
-  {
-    "fromPath": "/nfts/",
```

Comment:
Would we want to keep this redirect? I imagine the nft one above is more likely what was supposed to be removed.

Reply:
Gooood catch. Yes! exactly that, thanks. I've removed the wrong one. Fixing..